jakarta-eclipse/src/org/jakarta/jdt/servlet/CompleteServletAnnotationQuickFix.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+* Copyright (c) 2020, 2021 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.jakarta.jdt.servlet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.dom.IBinding;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.jakarta.codeAction.JavaCodeActionContext;
+import org.jakarta.codeAction.proposal.ChangeCorrectionProposal;
+import org.jakarta.codeAction.proposal.ModifyAnnotationProposal;
+
+/**
+ * QuickFix for fixing {@link ServletConstants#DIAGNOSTIC_CODE_MISSING_ATTRIBUTE} error
+ * by providing several code actions:
+ * 
+ * <ul>
+ * <li> Add the `value` attribute to the `@WebServlet` annotation
+ * <li> Add the `urlPatterns` attribute to the `@WebServlet` annotatio
+ * </ul>
```

Comment:
1) Missing the "n" on the second annotation
2) Add another description for the quick fixes for the `DIAGNOSTIC_CODE_DUPLICATE_ATTRIBUTES` diagnostic